src/main/resources/dbreplica/migration/V1.0.6__init_reportable_fields_db.sql

```diff
@@ -0,0 +1,55 @@
+
+drop table if exists reportable_task cascade;
+create table cft_task_db.reportable_task
+(
+  update_id               SERIAL NOT NULL,
```

Comment:
Is this supposed to be a copy of the task_history update_id?  In which case it doesn't make sense to have this as a serial?  Very easy for them to become out of sync and confuse everyone.

Reply:
As discussed with Ravi, this will be refactored RWA-2085